com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -14,29 +14,33 @@ The Decal Projector includes a Scene view representation of its bounds and proje
 
 * A box that describes the 3D size of the projector; the projector draws its decal on every Material inside the box.
 
-* An arrow that indicates the direction the projector faces.
+* An arrow that indicates the direction the projector faces. This arrow is on the pivot point.
 
 ![](Images/DecalProjector2.png)
 
-The decal Projector also includes two gizmos that add handles on every face for you to click and drag to alter the size of the projector's bounds.
+The decal Projector also includes three gizmos. The first two add handles on every face for you to click and drag to alter the size of the projector's bounds.
 
 |**Button**|**Gizmo**|**Description**|
 |-----|-----|-----|
-|![](Images/DecalProjector3.png)|**Scale**|Scales the decal with the projector box. This changes the UVs of the Material to match the size of the projector box. This stretches the decal.|
-|![](Images/DecalProjector4.png)|**Crop**|Crops the decal with the projector box. This changes the size of the projector box but not the UVs of the Material. This crops the decal.|
+|![](Images/DecalProjector3.png)|**Scale**|Scales the decal with the projector box. This changes the UVs of the Material to match the size of the projector box. This stretches the decal. The Pivot remains still.|
+|![](Images/DecalProjector4.png)|**Crop**|Crops the decal with the projector box. This changes the size of the projector box but not the UVs of the Material. This crops the decal. The Pivot remains still.|
+|![](Images/DecalProjector5.png)|**Pivot / UV**|Move the pivot without moving the projection box. This alter the transform position.<br />Also, set the UV used on the projected texture.|
+
+The color of the gizmos can be set up in the Preference window inside Color panel.
 
 ## Using the Inspector
 
 Using the Inspector allows you to change all of the Decal Projector properties, and lets you use numerical values for **Size**, **Tiling**, and **Offset**, which allows for greater precision than the click-and-drag gizmo method.
 
 ## Properties
 
-![](Images/DecalProjector5.png)
+![](Images/DecalProjector6.png)
 
 | **Property**            | **Description**                                              |
 | ----------------------- | ------------------------------------------------------------ |
 | **Size**                | The size of the projector influence box, and thus the decal along the projected plane. The projector scales the decal to match the **Width** (along the local x-axis) and **Height** (along the local y-axis) components of the **Size**. |
 | **Projection Depth**    | The depth of the projector influence box. The projector scales the decal to match **Projection Depth**. The Decal Projector component projects decals along the local z-axis. |
+| **Pivot**               | The offset position of the transform regarding the projection box. Adjusting X and Y allows to make rotation of the projected texture around a specific position, while Z allows to have a specific offset in depth. |
```

Comment:
The offset position of the transform regarding the projection box. To rotate the projected texture around a specific position, adjust the **X** and **Y** values. To set a depth offset for the projected texture, adjust the **Z** value.